segmentation/utils/data_utils.py

```diff
@@ -142,20 +156,53 @@ def load_imgs_from_dir(data_dir, img_sub_folder=None, fovs=None, imgs=None,
     if np.min(img_data) < 0:
         raise ValueError("Integer overflow from loading TIF image, try a larger dtype")
 
-    # remove .tif or .tiff from image name
-    img_names = [os.path.splitext(img)[0] for img in imgs]
-
```

Comment:
Following up on 118-120, if it's possible, it might be nice to add an else statement after lines 164-166 to handle this case of img_name extraction.